src/bin/tako-worker.rs

```diff
@@ -31,6 +31,9 @@ struct Opts {
     #[clap(long, default_value = "4000")]
     heartbeat: u32,
 
+    #[clap(long, default_value = "1000")]
```

Comment:
Could you please make this optional? We can either:
1) Disable it by default and enable it with some flag (HQ would automatically use this flag in auto allocation)
2) Enable it by default and provide an option to disable it

I would prefer the second option. @spirali?

Reply:
I also prefer (1). It is not so much important here in tako-worker (as it is now mostly testing platform), but it should be Option in worker settings